lib/auth/touchid/api.go

```diff
@@ -42,14 +44,29 @@ import (
 var (
 	ErrCredentialNotFound = errors.New("credential not found")
 	ErrNotAvailable       = errors.New("touch ID not available")
+
+	// PromptPlatformMessage is the message shown before Touch ID prompts.
+	PromptPlatformMessage = "Using platform authenticator, follow the OS prompt"
+	// PromptWriter is the writer used for prompt messages.
+	PromptWriter io.Writer = os.Stderr
```

Comment:
Do these variables need to be public?
Why do not make `PromptPlatformMessage` a const?

Reply:
I made them public so they can be changed/customized from caller packages. We don't need to tweak those messages today, but we do so for some other prompt messages, so it's not a big stretch.